src/cmr/sizing/formats.clj

```diff
@@ -29,20 +35,31 @@
       (get-in [:umm :SizeEstimation :AverageSizeOfGranulesSampled])
       read-number))
 
-(defn- get-avg-compression-rate-ascii
-  "Gets SizeEstimation value for AvgCompressionRateASCII and parses it to a number."
-  [variable]
-  (-> variable
-      (get-in [:umm :SizeEstimation :AvgCompressionRateASCII])
-      read-number))
-
 (defn- get-avg-compression-rate-netcdf4
   "Gets SizeEstimation value for AvgCompressionRateNetCDF4 and parses it to a number."
   [variable]
   (-> variable
       (get-in [:umm :SizeEstimation :AvgCompressionRateNetCDF4])
       read-number))
 
+(defn- get-avg-fill-value-digit-number
+  "Get the average digit number for each value of FillValues in the variable.
+  For example, FillValues containing [{:Value 1} {:Value 100}], returns (1+3)/2=2."
+  [variable]
+  (let [fvs (get-in variable [:umm :FillValues])
+        fvs-digit-numbers (map #(count (str (:Value %))) fvs)]
+    (/ (reduce + fvs-digit-numbers) (count fvs-digit-numbers))))
```

Comment:
Since you use this a couple of times it might be worth pulling out into an average function.